thirdparty/onnx/CMakeLists.txt

```diff
@@ -51,13 +51,8 @@ if(WIN32)
     if(MINGW)
         target_compile_definitions(onnx PRIVATE OPTIONAL=OPTIONAL_PLACEHOLDER)
     endif()
-endif()
-
-# WA to support old compiler version, removes unused header which may block build
-if(CMAKE_COMPILER_IS_GNUCXX AND LINUX AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 7.0)
-    file(READ onnx/onnx/defs/parser.cc CONTENT)
-    string(REPLACE "#include <string_view>" "" CONTENT "${CONTENT}")
-    file(WRITE onnx/onnx/defs/parser.cc "${CONTENT}")
+else()
+    target_compile_features(onnx PRIVATE cxx_std_17)
```

Comment:
looks like we can set it unconditionally because for Windows we do the same.

Reply:
looks like yes, I thought we will need to use stdc++fs as additional option for linux, but it isn't necessary for our ci environment